pkgs/development/tools/build-managers/rebar3/default.nix

```diff
@@ -39,6 +39,8 @@ let
       HOME=. escript bootstrap
     '';
 
+    patches = if (stdenv.isDarwin && stdenv.isAarch64) then [ ./m1-tests-skip.patch ] else [];
```

Comment:
```suggestion
    patches = lib.optionals (stdenv.isDarwin && stdenv.isAarch64) [ ./m1-tests-skip.patch ];
```

Reply:
I can, but I don't quite understand how this differs if it's still an if underneath? Does it somehow affect what ends up in the store and using `lib.optionals` makes sure that someone won't lose the file even if they're not on an M1?

Reply:
it is a conditional, yes.
It's a little more idiomatic in nixpkgs, it absolutely does not change the logic with what you had.

Reply:
Used optionals and added a comment about removing then on next release